app/models/medium.rb

```diff
@@ -0,0 +1,5 @@
+class Medium < ApplicationRecord
+  MEDIA_TYPES = ['In Person', 'Phone Call', 'Text Message', 'Voice Message', 'Social Media', 'Gaming', 'Email', 'Meme']
+
+  belongs_to :user
```

Comment:
Are we missing a validation here regarding the media types?  The same way we are doing for the contact relationship?

Reply:
Done ✅